modules/oe_content_person/oe_content_person.info.yml

```diff
@@ -0,0 +1,59 @@
+name: OpenEuropa Person Content
+description: Contains the EC corporate Person content type.
+package: OpenEuropa
+type: module
+core: 8.x
+
+dependencies:
+  - field_group:field_group
+  - oe_content:oe_content
+  - oe_content:oe_content_entity_contact
+  - oe_content:oe_content_departments_field
+  - oe_content:oe_content_timeline_field
+  - entity_reference_revisions:entity_reference_revisions
```

Comment:
Don't we miss these dependencies still (coming from the default entity form that we export)?

```
    - inline_entity_form
    - path
    - text
```

Reply:
`drupal:text` dependency exists in drupal:node module.

Reply:
I will make a test and get back with the correct list of dependencies

Reply:
I created a small functional test on my local to check which dependencies it complains for, the list was amended